### PR TITLE
Wanderer Staking Pledge

### PR DESCRIPTION
**Pull request by Wanderer-Cry**

### CODE REVIEW

File: Wanderer Staking/README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Agoric Validator Pledge
+
```
Comment (jeetraut) [inline Wanderer Staking/README.md:2]:
```suggestion
[Wanderer Staking Explorer Link](https://main.explorer.agoric.net/validator/agoricvaloper1kxekyxvywtqexan9m9uju69ger67vefeeuhgrz)
```

**Review by jeetraut — CHANGES_REQUESTED** [review]
Add explorer link

File: Wanderer Staking/README.md (hunk 1 above)
Comment (Wanderer-Cry) [reply Wanderer Staking/README.md:2]: Agreed.

**Comment by Wanderer-Cry** [comment]
Added. Thank you

**Review by Wanderer-Cry — COMMENTED** [review]
Agreed

**Comment by Wanderer-Cry** [comment]
> Add explorer link

Added, thanks

**Pull request merged**